README.md

```diff
@@ -78,6 +78,9 @@ This package contains customizable rules for identifying references to banned AP
 
 This package contains rules to help library authors monitoring change to their public APIs. [More info about rules in this package](src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md)
 
+For instructions on using this analyzer, see [Instructions]((https://github.com/dotnet/roslyn-analyzers/blob/master/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md). Also see this [blog post](https://www.natmarchand.fr/prevent-breaking-changes-apis-roslyn/).
```

Comment:
It seems weird to reference other websites. If the instructions are not clear enough, wouldn't it be better to actually improve them?

Reply:
I agree with @Evangelink. Please revert the reference to external website. It would be fine to add more documentation to `PublicApiAnalyzers.Help.md`.

Reply:
I've been thinking about it, and yes I agree.
But I'd be essentially plagiarising the content from that blog, which is probably worse.

Reply:
No need for plagiarism. Just ask yourself why you felt that blog post was clearer, then work out where the existing documentation falls short and fill in the gaps. No reason you can't make the docs better than the blog post :)

Reply:
I agree with @drewnoakes. @RussKie feel free to update the PR with additions to `PublicApiAnalyzers.Help.md`. Thanks!

Reply:
Fair enough.
I don't enough free time atm to update it, so I just remove the external link.